tests/integration/ha_tests/helpers.py

```diff
@@ -215,6 +215,26 @@ def cut_network_from_unit(machine_name: str) -> None:
     subprocess.check_call(cut_network_command.split())
 
 
+def cut_network_from_unit_without_ip_change(machine_name: str) -> None:
+    """Cut network from a lxc container (without causing the change of the unit IP address).
+
+    Args:
+        machine_name: lxc container hostname
+    """
+    override_command = f"lxc config device override {machine_name} eth0"
+    try:
+        subprocess.check_call(override_command.split())
+    except subprocess.CalledProcessError:
+        # Ignore if the interface was already overridden.
+        pass
+    limit_set_command = f"lxc config device set {machine_name} eth0 limits.egress=0kbit"
+    subprocess.check_call(limit_set_command.split())
+    limit_set_command = f"lxc config device set {machine_name} eth0 limits.ingress=1kbit"
```

Comment:
maybe
```suggestion
    limit_set_command = f"lxc config device set {machine_name} eth0 limits.ingress=0kbit"
```

Reply:
For ingress, 0kbit is not allowed. 1kbit is the minimum allowed value. I did some manual tests (along with the automated test from the PR) and the connections to the database or Patroni fail at the other side (other unit or client application) as the response to the connections is not returned due.